test/bin/mirror_registry.sh

```diff
@@ -222,9 +251,9 @@ EOF
     fi
 
     # See https://docs.projectquay.io/deploy_quay.html#preparing-local-storage
-    if [ ! -d "${MIRROR_REGISTRY_DIR}/storage" ] ; then
-        mkdir -p "${MIRROR_REGISTRY_DIR}/storage"
-        setfacl -m u:1001:-wx "${MIRROR_REGISTRY_DIR}/storage"
+    if [ ! -d "${QUAY_STORAGE_DIR}" ] ; then
+        mkdir -p "${QUAY_STORAGE_DIR}"
+        setfacl -m u:1001:-wx "${QUAY_STORAGE_DIR}"
```

Comment:
Can we run quay container as `"$(id -u):$(id -g)"`?

Reply:
This does not work because the container performs operations on /etc. It then becomes user 1001
```
Running init script '/quay-registry/conf/init/certs_install.sh'
p11-kit: couldn't create file: /etc/pki/ca-trust/extracted/openssl/ca-bundle.trust.crt: Unknown error 13
p11-kit: couldn't create file: /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem: Unknown error 13
p11-kit: couldn't create file: /etc/pki/ca-trust/extracted/pem/email-ca-bundle.pem: Unknown error 13
p11-kit: couldn't create file: /etc/pki/ca-trust/extracted/pem/objsign-ca-bundle.pem: Unknown error 13
p11-kit: couldn't create file: /etc/pki/ca-trust/extracted/java/cacerts: Unknown error 13
p11-kit: couldn't create file: /etc/pki/ca-trust/extracted/edk2/cacerts.bin: Unknown error 13
```

Reply:
Let me try another approach?